src/commands/add.ts

```diff
@@ -161,12 +161,18 @@ const addUI = async (command: Command): Promise<void> => {
   packageJson.scripts = {
     ...packageJson.scripts,
     [`ui:${appName}:start`]: `parcel src/${appName}/index.html --out-dir build/${appName}/debug`,
-    [`ui:${appName}:build`]: `parcel build src/${appName}/index.html --out-dir src/${appName}/release --public-url ./`,
   }
   if (createRuntimeSupport(pondVersion)) {
     packageJson.scripts = {
       ...packageJson.scripts,
       [`ui:${appName}:package`]: `ax apps package src/${appName}/ax-manifest.yml`,
+      [`ui:${appName}:build`]: `parcel build src/${appName}/index.html --out-dir src/${appName}/release --public-url ./`,
+    }
+  } else {
```

Comment:
I guess this warrants a comment.
It took me a while to see the difference in `src/` and `build/`